src/js/edu-benefits/reducers/uiState/index.js

```diff
@@ -0,0 +1,182 @@
+import _ from 'lodash/fp';
+
+import {
+  UPDATE_COMPLETED_STATUS,
+  UPDATE_INCOMPLETE_STATUS,
+  UPDATE_REVIEW_STATUS,
+  UPDATE_VERIFIED_STATUS,
+  UPDATE_SUBMISSION_STATUS,
+  UPDATE_SUBMISSION_ID,
+  UPDATE_SUBMISSION_TIMESTAMP
+} from '../../actions';
+
+const ui = {
+  submission: {
+    status: false,
+    errorMessage: false,
+    id: false,
+    timestamp: false
+  },
+  sections: {
```

Comment:
And also from this spreadsheet (might be more up to date): https://github.com/department-of-veterans-affairs/edu-benefits-team/blob/master/design-documents/Proposed%20question%20flow.xlsx